README.md

```diff
@@ -1,21 +1,61 @@
 Zinc
 ====
 
-Zinc is an incremental compilation module of sbt with the sbt-agnostic API.
+Zinc is the incremental compiler for Scala. Most Scala developers use it daily
+without noticing -- it's embedded in key build tools like sbt, CBT and pants.
```

Comment:
The old republished "Zinc" *was* embedded into sbt, etc. the new Zinc is a standalone incremental compiler library intending to be used by everyone. sbt 1 *will* use it, and I think Pants is committed to using it, but I don't know of others yet.